### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by minikin**
- Add pull_request_template

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Description
+
+Please include a summary of the changes and the related issue.
+Please also include relevant motivation and context.
+List any dependencies that are required for this change.
+
+Fixes # (issue)
```
Comment (Mr-Leshiy) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:7]: I would make field as an optional

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 
+Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+**Test Configuration**:
```
Comment (Mr-Leshiy) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:26]: Is it mandatory or optional field ?

**Review by saibatizoku — APPROVED** [review]
LGTM. I'm with @Mr-Leshiy , and I think that the template should indicate what is and isn't optional, to guide the user.

**Review by kukkok3 — COMMENTED** [review]
LGTM

**Comment by minikin** [comment]
@stevenj @cong-or @cameron1024 @dkijania @nicopado @FelipeRosa 
Can you have a look, please?

**Review by cameron1024 — APPROVED** [review]

**Comment by dkijania** [comment]
Looking great @minikin . I would only address @Mr-Leshiy remarks as I share the same concerns

**Review by Mr-Leshiy — APPROVED** [review]

**Pull request merged**